src/main/java/org/folio/mr/controller/MediatedRequestActionsController.java

```diff
@@ -79,21 +82,25 @@ private static ConfirmItemArrivalResponse buildConfirmItemArrivalResponse(Mediat
   }
 
   @Override
-  public ResponseEntity<SendItemInTransitResponse> sendItemInTransit(
-    SendItemInTransitRequest request) {
-    
+  public ResponseEntity<SendItemInTransitResponse> sendItemInTransit(SendItemInTransitRequest request) {
     log.info("sendItemInTransit:: request={}", request);
     MediatedRequest mediatedRequest = actionsService.sendItemInTransit(request.getItemBarcode());
 
-    return ResponseEntity.ok(buildSendItemInTransitResponse(mediatedRequest));
+    return ResponseEntity.ok(buildSendItemInTransitResponse(mediatedRequest, getActionDate(mediatedRequest)));
+  }
+
+  private Date getActionDate(MediatedRequest request) {
```

Comment:
Let's rename it to logAction() or logActionAndGetActionDate(), otherwise it is confusing - it looks like a `get` method but quietly writes things into the DB.